policyengine_us/parameters/gov/irs/credits/retirement_saving/rate/head_of_household.yaml

```diff
@@ -0,0 +1,43 @@
+description: head of household saver's credit threshold and rate
+metadata:
+  threshold_unit: currency_USD
+  amount_unit: /1
+  threshold_period: year
+  type: single_amount
+  label: head of household saver's credit rate
+  reference: 
+    - title: IRS | Retirement Savings Contributions Credit (Saver’s Credit)
+      href: https://www.irs.gov/retirement-plans/plan-participant-employee/retirement-savings-contributions-savers-credit
+    - title: 2022 Publication 590-A
+      href: https://www.irs.gov/pub/irs-pdf/p590a.pdf#page=45
+    - title: Form 8880 - 2023 Credit for Qualified Retirement Savings Contributions
```

Comment:
add 2024 values (see references for other 2024 values like the core tax brackets)

Reply:
[2024 values have not been changed yet](https://www.irs.gov/retirement-plans/plan-participant-employee/retirement-savings-contributions-savers-credit)

Reply:
Huh, and also not in the IRS publication: https://www.irs.gov/pub/irs-drop/rp-23-34.pdf#page=14

Thanks for checking.